packages/design-system/src/components/Icons/EmailIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import type * as React from 'react';
+import { t } from '../i18n';
+import { SvgIcon, IconCommonProps } from './SvgIcon';
+
+const defaultProps = {
+  className: '',
+  viewBox: '0 0 32 32',
```

Comment:
Nothing wrong here, just a callout that it's interesting the `viewBox` is the same for all new icons. I wonder if there's an opportunity to standardize our icons in the future?